Temmietayo/All-Recipes
# Ch recipe details template 169810647

#### What does this PR do?
Add the template for user to see recipe details .

#### Description of Task to be completed?
A page to show details of a recipe

#### How should this be manually tested?

- clone the repo

- cd into the template directory

- open the recipelist.html with any browser of your choice

- these template should open in that browser
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[169810647](https://www.pivotaltracker.com/story/show/169810647)
